types/chai-as-promised/chai-as-promised-tests.ts

```diff
@@ -1,4 +1,5 @@
 import * as chai from "chai";
+import "chai/register-should.js";
```

Comment:
This makes me wonder why `chai` hasn't used export maps to deal with this. chai v5 is ESM only, so, it's somewhat surprising. But, if this is how it is, so be it.